<launch>
	<!-- vim: set ft=xml noet : -->
	<arg name="fcu_url" default="serial:///dev/ttyUSB0:921600" />
	<arg name="gcs_url" default="" />
	<arg name="tgt_system" default="1" />
	<arg name="tgt_component" default="1" />
	<arg name="log_output" default="screen" />

	<include file="$(find rqt_gcs)/launch/node_machine.launch">
		<arg name="pluginlists_yaml" value="$(find mavros)/launch/px4_pluginlists.yaml" />
		<arg name="config_yaml" value="$(find mavros)/launch/px4_config.yaml" />
		<arg name="fcu_url" value="$(arg fcu_url)" />
		<arg name="gcs_url" value="$(arg gcs_url)" />
		<arg name="tgt_system" value="$(arg tgt_system)" />
		<arg name="tgt_component" value="$(arg tgt_component)" />
		<arg name="log_output" value="$(arg log_output)" />
	</include>
	<!-- <node pkg="machine_vision" type="stereo_driver" name="stereo_driver" required="true"/>
	<node pkg="machine_vision" type="object_detect" name="object_detection" required="true"/>
        <node pkg="machine_vision" type="object_avoidance" name="object_avoidance" required="true" />
        <node pkg="stereo_image_proc" type="stereo_image_proc" name="stereo_image_proc" required="true" ns="stereo_cam"/>
	<param name="image_transport" type="string" value="compressed" /> 
        
        -->

</launch>
